### PR TITLE
[DEV-224374] Add basic contributing guide to example apps

### PR DESCRIPTION
**Pull request by renatodelpupo**
### Jira Issue

https://trustly.atlassian.net/browse/DEV-224374

---

### Description

Improve our repository standards by adding a Pull Request Template and Contributing instructions.

---

### Relevant Commits

- [Create Pull Request Template](https://github.com/TrustlyInc/trustly-webview-example/commit/bd498562f8ca915b919bc8dcc3bb9553e66044f5)
- [Add Contributing instructions to README](https://github.com/TrustlyInc/trustly-webview-example/commit/396ae96dee6f5e44fb264d4cd7ca39f7b25cf7e6)

---

### Requirements

- [x] Changes were properly tested (attach evidence if applicable)
- [x] Repository's code-style/linting compliant

---

### Evidence

DEV-224374  Basic contributing guide to example apps

---

### Additional Information

None.

### CODE REVIEW

**Review by andresguedes — APPROVED** [review]
👍🏽

**Review by marcosrivereto — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+### Jira Issue
```
Comment (lukevance) [inline .github/pull_request_template.md:1]: Can we move this down a bit or will it break the Jira integration? Although at this point most PRs will have an associated Jira, this is an open source repo and we should prepare for contributors outside of Trustly.

File: README.md [hunk 2]
```diff
@@ -14,4 +14,10 @@ The `ios-app` directory contains a simple Swift app that renders a Webview of a
 
 ## The Android App
 
-The `android-app` directory contains a simple Kotlin app that renders a Webview of a given URL. With the above React app running, the Android app can be pointed to the resulting URL to demonstrate the webview and secure-in-app-browser functionality required to successfully integrate the two applications. Additional instructions can be found in that directory's [ReadMe](./android-app/README.md).
\ No newline at end of file
+The `android-app` directory contains a simple Kotlin app that renders a Webview of a given URL. With the above React app running, the Android app can be pointed to the resulting URL to demonstrate the webview and secure-in-app-browser functionality required to successfully integrate the two applications. Additional instructions can be found in that directory's [ReadMe](./android-app/README.md).
+
+# Contributing
```
Comment (lukevance) [inline README.md:19]: 🙌🏻 perfect! TY

File: .github/pull_request_template.md (hunk 1 above)
Comment (renatodelpupo) [reply .github/pull_request_template.md:1]: Hi, @lukevance. Yeah, good point!

I can move it down, flag it as optional, or even "merge" it with the Additional Information topic.

Maybe something like:

_(Insert any additional information related to this pull request, as more context or Jira ticket. Leave it as **"None"** or **"Not applicable"** if you have nothing to add)_

I don't think that the integration will break, but I also don't have a previous experience with this.

Comment (lukevance) [reply .github/pull_request_template.md:1]: I like the idea of merging it, as long as it is obvious for our team where the JIRA should go but not confusing to external contributors, that is the goal 👍🏻

Comment (renatodelpupo) [reply .github/pull_request_template.md]: Updates applied!

**Review by lukevance — APPROVED** [review]
All set! Thanks for the revisions!

**Pull request merged**